catalogue/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -240,6 +239,77 @@ const WorkDetails: FunctionComponent<Props> = ({ work }: Props) => {
 
   const Content = () => (
     <>
+      {showHoldingsOnWork && holdings.length > 0 && (
+        <WorkDetailsSection headingText="Holdings" isInArchive={isInArchive}>
+          {holdings.map((holding, i) => (
+            <div key={i}>
+              {holding.enumeration.length > 0 && (
+                <>
+                  <h3>Enumeration</h3>
+                  <ul>
+                    {holding.enumeration.map((e, i) => (
```

Comment:
I know we're just trying stuff out, but if we have better keys we should use them to avoid forgetting to clear this up.